nuxeo-runtime/nuxeo-runtime-reload/src/main/java/org/nuxeo/runtime/reload/ReloadResult.java

```diff
@@ -49,6 +51,14 @@ public ReloadResult() {
         return deployedBundles;
     }
 
+    public Stream<Bundle> deployedBundlesAsStream() {
```

Comment:
missing since annotation

Reply:
The @since annotation is on the class, is it not enough for such developments ?

Reply:
Yes it is, i did not realize the whole class was new on this PR